src/main/java/com/devathon/preguntonic/model/Room.java

```diff
@@ -0,0 +1,29 @@
+/*
+ *
+ * Copyright (c) 2024 Devathon., All Rights Reserved.
+ *
+ */
+package com.devathon.preguntonic.model;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Room {
+
+  private UUID id;
```

Comment:
I know the following comments are late.😞 

Since this class is ephemeral, do you think we should use this field?

Reply:
Yes! We should manage a identifier to get this object from a local storage (a hash map i think)